pyproject.toml

```diff
@@ -54,6 +55,7 @@ dbfread = {version = "*", optional = true}  # groups = ["user", "dev"]
 pydicom = {version = ">=2.0.0", optional = true}  # groups = ["user", "dev", "dicom"]
 pynetdicom = {version = "*", optional = true}  # groups = ["user", "dev", "dicom"]
 pylibjpeg-libjpeg = {version = "*", optional = true}  # groups = ["user", "dev", "dicom"]
+immutables = {version = "*", optional = true}  # groups = ["user", "dev", "dicom"]
```

Comment:
When adding this, can you also run the following script:

https://github.com/pymedphys/pymedphys/blob/master/scripts/propagate-extras.py

Reply:
when I went to run propagate-extras.py:
``python scripts/propagate-extras.py``
It complained about a missing package, so I did
poetry install -E dev
and I noticed the following:
- Updating numpy (1.19.1 -> 1.18.5)
while all other updates were more like:
 - Updating pylint (2.5.3 -> 2.6.0)
so... lower semantic version first, newer/higher semantic version second.

On the plus side, the poetry install -E dev fins and running propagate-extras.py is done, so... pushing